### DISCUSSION TITLE
Option for disabling trailing slash for some routes

### DISCUSSION THREAD

**Discussion by amirhbakan** [vercel/next.js#33927] [Ideas]
### Describe the feature you'd like to request

we are migrating a legacy website to a new website. they have a weird rule, that all pages have trailing slash except 2 of them. for SEO, we should not change trailing slash rules, how can I disable trailing slash just for some pages?

### Describe the solution you'd like

all-pages -> all-pages/
some-pages/ -> some-pages
some-pages -> some-pages

### Describe alternatives you've considered

if I can disable the trailing slash feature completely is also good, I can handle this at the infra structure level.

**Comment by rteuscher** [comment]
We have a similar need. We have a Ghost blog set up as a subdirectory that requires a trailing slash. The rest of our product does not play nice with trailing slashes.

**Comment by robertn702** [comment]
@rteuscher Did you ever find a solution for this? I have the exact same situation and am running into the same problem.
